package.json

```diff
@@ -4,9 +4,10 @@
   "description": "bc-nc-news",
   "main": "index.js",
   "scripts": {
+    "start": "PGDATABASE=nc_news_test nodemon listen.js",
     "setup-dbs": "psql -f ./db/setup.sql",
-    "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "seed": "PGDATABASE=nc_news_test node ./db/seeds/run-seed.js",
+    "test": "PGDATABASE=nc_news_test jest",
```

Comment:
You should be using dotenv to store your database names inside .env files that won't get pushed to github. Pushing database names in the package.json is not good practice